test/tests/functional/pbs_hook_timeout.py

```diff
@@ -71,8 +70,14 @@ def setUp(self):
         self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostB)
 
         self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostC)
+        for mom in self.moms.values():
+            self.server.expect(NODE, {'state': 'free'}, id=mom.shortname)
+        """
+        self.server.expect(VNODE, {'state=free': 3}, op=EQ, count=True,
+                           max_attempts=10, interval=2)
+        """
 
-    @skipOnCpuSet
+    @timeout(600)
```

Comment:
isn't 600 the default?

Reply:
It wasn't intentional, added it by mistake. I have now reverted back the file to orignal test